dockers/docker-syncd-mlnx-rpc/Dockerfile

```diff
@@ -44,5 +45,4 @@ COPY ptf_nn_agent.conf /etc/supervisor/conf.d/ptf_nn_agent.conf
 
 ENTRYPOINT service rsyslog start \
     && service syncd start       \
-    && while [ "x$( ifconfig Ethernet12 | grep RUNNING )" = 'x' ] ; do sleep 1 ; done \
```

Comment:
so the PR is not based on the latest master. better to generate PR based on latest master.